docs/04-Bar-Chart.md

```diff
@@ -49,6 +49,7 @@ borderSkipped | `String or Array<String>` | Which edge to skip drawing the borde
 hoverBackgroundColor | `Color or Array<Color>` | Bar background color when hovered
 hoverBorderColor | `Color or Array<Color>` | Bar border color when hovered
 hoverBorderWidth | `Number or Array<Number>` | Border width of bar when hovered
+stack | `String` | For grouping stacked bars
```

Comment:
I think that's a good call then. You're right that the numbers could imply an order to the stacks.

I think `stack` is ok. Maybe @simonbrunel has some thoughts on this